shopinvader/models/shopinvader_category.py

```diff
@@ -42,7 +42,12 @@ class ShopinvaderCategory(models.Model):
         compute="_compute_child_category",
     )
     level = fields.Integer(compute="_compute_level")
-    active = fields.Boolean(default=True)
+    active = fields.Boolean(
```

Comment:
this could be moved to shopinvader.binding

Reply:
Oh but not all binded records will have an `active` field.
Take this one for example, `product.category` doesn't have active in core. It's added by shopinvader (probably a mistake - I kept it not to break anything).

Reply:
What @simahawk wanted to say is that with this implementation, you need to put this declaration in every shopinvader.x model.

As they inherit from shopinvader.binding, it is more convenient to put it there.